app/assets/stylesheets/styles/components/guides/components/_guide_creation_nav.css.scss

```diff
@@ -0,0 +1,84 @@
+.guide-creation-nav {
+  $guide-nav-color: #5E9546;
+  background: $guide-nav-color;
+
+  /* the ul containing the links */
+  ul {
+    padding-top: 5px;
+    text-align: center;
+    list-style: none;
+    white-space: nowrap;
+  }
+
+  li {
+    position: relative;
+    display: inline-block;
+  }
+
+  li a {
+    color: white;
+    font-size: 1.25em;
+    font-weight: 600;
+    vertical-align: middle;
+    color: rgba(255,255,255,0.7);
+    padding: 10px 10px 10px 35px;
+
+    &.disabled {
+      pointer-events: none;
+    }
+
+    &:hover {
+      color: rgba(255,255,255,0.9);
+    }
+
+    &::before {
+      position: absolute;
+      top: 10px;
+      left: 10px;
+      $size: 12px;
+      width: $size; height: $size;
+      content: "";
+      @include box-shadow(
+          0 0 0 2px $guide-nav-color,
+          0 0 0 4px #D9E9D2);
+      border-radius: 50%;
+    }
+
+    &.check::after {
+      @include manual-font-awesome('\f00c');
+      position: absolute;
+      top: 4px;
+      left: 8px;
+      font-size: 22px;
+      color: white;
+    }
+
+    &:active, &.active {
+      color: rgba(255,255,255,1);
+
+      // get rid of check
+      &::after {
+        display: none;
+      }
+    }
+
+    &.active::before {
+      background: #D9E9D2;
```

Comment:
Color literals like `#D9E9D2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#D9E9D2` should be written as `#d9e9d2`